notesapi/v1/tests/test_views.py

```diff
@@ -54,7 +55,7 @@ def setUp(self):
             "tags": ["pink", "lady"]
         }
 
-    def _create_annotation(self, **kwargs):
+    def _create_annotation(self, check_status=True, **kwargs):
```

Comment:
Instead of doing this, it would be cleaner to add parameters `expected_status=status.HTTP_201_CREATED, error_message=None`. And in the body do:

```
self.assertEqual(response.status_code, expected_status_code)
if error_message:
  self.assertEqual(response.data['error_msg'], error_message)

```